### DISCUSSION TITLE
Migrate website from WordPress to Statamic v3

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#4524] [Q&A] [answered]
On the old Statamic forum there is this thread:

[Migrate website from WordPress to Statamic v3](https://statamic.com/forum/5053-migrate-website-from-wordpress-to-statamic-v3)

Are there any news on updating the https://github.com/statamic/wordpress-to-statamic-exporter to Statamic v3?

We have to migrate a site soon (with ca. 1500 blog entries, some 200 comments, and ca. 3850 media files used in the blog posts) and wonder whether to got the WP -> v2 -> v3 route or use Corcel? We also have to estimate working hours, phew ...

So maybe if you have any experience and any opinions on that that would be extremly helpful. :-)

Many thanks in advance!

Jens Olaf

**Comment by edalzell** [comment, marked as answer]
I'd use Corcel.

**Reply by jensolafkoch** [reply]
Thanks, Erin!
